wsdl/ver10/schema/onvif.xsd

```diff
@@ -3828,6 +3828,22 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 		</xs:restriction>
 	</xs:simpleType>
 	<!--===============================-->
+	<xs:complexType name="EditableUserLevel">
+		<xs:sequence>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Name of the editable user level.</xs:documentation>
```

Comment:
Currently ONVIF have Administrator, Operator, User, Anonymous, Extended User levels defined and the access permissions for these user levels also pre-defined. So Name parameter in EditableUserLevel  should not conflict with any of the existing ONVIF Pre-Defined user levels so that the behavior of the ONVIf pre-defined user levels should not be changed.

Reply:
In my vision, they do not conflic, because to use the EditableUserLevels the Userlevel must always be set to extended. But I understand that this may lead to tricky names. So, let's gather the opinion of other members.